docs/tutorials/scilla-tutorials/nft-collection/public-metadata.md

```diff
@@ -0,0 +1,171 @@
+---
+sidebar_position: 4
+---
+
+# Creating metadata for a token
+
+The below is an example of what metadata for a token should be presented. Read the document for [ZRC-7 nonfungible token metadata standardisation](https://github.com/Zilliqa/ZRC/blob/master/zrcs/zrc-7.md) for the full standard specification.
+
+Take note of the metadata 'resource' being the previously uploaded resource file location.
+
+```json {3}
+{
+  "name": "Creature #101",
+  "resources": [
+    { "uri": "ipfs://QmZILGa7zXUbixvYJpgkRkaSCYEBtSwgVtfzkoD3YkNsE1" }
+  ],
+  "attributes": [
+    {
+      "trait_type": "Background",
+      "value": "Black"
+    },
+    {
+      "trait_type": "Eyes",
+      "value": "Big"
+    },
+    {
+      "trait_type": "Mouth",
+      "value": "Grin"
+    }
+  ]
+}
+```
+
+## Making metadata public
+
+Previously we uploaded our image resource, which is attached on our metadata json object on line 3. This metadata json also needs to be uploaded so we can use the metadata link on the token.
+
+By having the image wrapped inside the metadata, it allows us to pass both the image and some richer details of the token, instead of passing just the image alone.
+
+## base_uri vs token_uri
+
+Now with our image and metadata uploaded, we can decide if we want an API to serve the metadata, or if we want to directly embed it onto the token.
+
+The advantage of having an API serve the metadata is that a developer can control what the API is exposing. If files on IPFS need to change, the API can handle showing the correct file or data whilst hiding the data for tokens yet to be minted. In the case of when files need to change whilst embedded them onto the token, this is not possible as when a token_uri is set, the value is then immutable.
+
+The developer can set base_uri to an API they control. An example is when ```initial_base_uri``` is set to ```www.api.example.com/``` and token_id is ```1```, ecosystem partners will query ```www.api.example.com/1```.
+
+The advantage of embedding the file onto the token is that it's quick and easy with no API required. But then is immutable for further changes. It is recommended to use a solution like decentralised storage to ensure your content isn't taken offline at any stage if opting for a token_uri implementation.
```

Comment:
As it is about the data URL scheme, provide examples.

Reply:
Can you elaborate a bit further to what you are wanting here, I don't fully understand.

Reply:
I've elaborated more here: https://github.com/Zilliqa/ZRC/pull/179

Please use this link `https://github.com/Zilliqa/ZRC/blob/main/zrcs/zrc-7.md#i-what-is-metadata-and-token-uri` to explain metadata and token URIs.

something like:
```
For more details please visit: https://github.com/Zilliqa/ZRC/blob/main/zrcs/zrc-7.md#i-what-is-metadata-and-token-uri
```

Reply:
Note https://github.com/Zilliqa/Scilla-Cookbook/pull/31#issuecomment-1058689970